Unit tests broken for streams storage

This kind of blocks #446 
Steps to reproduce it (run unit tests of streams storage variant) :
```
mvn clean install -DskipTests -Pstreams
mvn verify -pl storage/streams/ -Dtest=SerdeMixTest#testVersions -Pstreams
```
```
[ERROR]   SerdeMixTest.testVersions:63 » RestClient No artifact with ID 'GlobalId: 0' wa...
[ERROR]   SerdeMixTest.testVersions:63 » RestClient No artifact with ID 'GlobalId: 65536...
```

--- a/app/src/test/java/io/apicurio/registry/SerdeMixTest.java
+++ b/app/src/test/java/io/apicurio/registry/SerdeMixTest.java
@@ -57,9 +57,10 @@ public class SerdeMixTest extends AbstractResourceTestBase {
         Schema schema = new Schema.Parser().parse("{\"type\":\"record\",\"name\":\"myrecord5\",\"fields\":[{\"name\":\"bar\",\"type\":\"string\"}]}");
         int id = confClient.register(subject, schema);
         confClient.reset();
-        
+
         this.waitForArtifact(subject);
 
+        this.waitForGlobalId(id);
         Schema schema2 = confClient.getById(id);
         Assertions.assertNotNull(schema2);
 

